components/portal/timeout/controllers.js

```diff
@@ -103,7 +103,7 @@ define(['angular'], function(angular) {
      * Redirect to end the session;
      */
     function redirect() {
-      $window.location.replace(MISC_URLS.loginURL);
+      $window.location.replace(MISC_URLS.logoutURL);
```

Comment:
Handle case where logoutURL not configured, or figure it's always configured and so can be relied upon?

Thinking about this slightly longer, the thing to do might be to predicate the your-session-will-end-soon dialog feature on `MISC_URLS.logoutURL` being set, since if it's not set the front end can't do much reasonable on session timeout.

Reply:
It seems the documentation was the only thing requiring that, so thanks for reminding me to update the documentation.

I'm 50/50 on checking the logoutURL, if we do I'm thinking we do it in `portalShibbolethService.shibServiceActivated()` As of right now, it's just treated as a misconfiguration.